### PR TITLE
Switch to i18n-next

### PR DESCRIPTION
**Pull request by Jack-Works**
tested in twitter & storybook.

### CODE REVIEW

File: src/components/InjectedComponents/ImmersiveSetup/SetupStepper.tsx [hunk 1]
```diff
@@ -59,11 +59,14 @@ export interface ImmersiveSetupStepperUIProps {
...
 export function ImmersiveSetupStepperUI(props: ImmersiveSetupStepperUIProps) {
+    const { t } = useI18N()
     const classes = useStyles()
     const steps = getSteps()
     const activeStep = props.currentStep
     const [, inputRef] = useCapturedInput(props.onUsernameChange)
 
+    const ERROR_TEXT = t('immersive_setup_no_bio_got')
+
     // const backButton = (
```
Comment (septs) [inline src/components/InjectedComponents/ImmersiveSetup/SetupStepper.tsx]: remove unused code?

File: src/components/InjectedComponents/PostDialog.tsx [hunk 2]
```diff
@@ -301,20 +303,20 @@ export function PostDialog(props: PostDialogProps) {
...
-                        warningText: geti18nString('additional_post_box__steganography_post_failed'),
+                        warningText: t('additional_post_box__steganography_post_failed'),
                     })
                 } else {
-                    let text = geti18nString('additional_post_box__encrypted_post_pre', encrypted)
+                    let text = t('additional_post_box__encrypted_post_pre', { encrypted })
                     if (metadata.ok) {
-                        if (getCurrentLanguage() === 'zh') {
+                        if (i18n.language.includes('zh')) {
                             text =
```
Comment (septs) [inline src/components/InjectedComponents/PostDialog.tsx]: The text write into messages file?

File: src/social-network-provider/twitter.com/ui/index.ts [hunk 3]
```diff
@@ -19,14 +19,12 @@ export const instanceOfTwitterUI = defineSocialNetworkUI({
     ...twitterUICustomUI,
     i18nOverwrite: {
```
Comment (septs) [inline src/social-network-provider/twitter.com/ui/index.ts]: move the text into messages file.

File: src/components/InjectedComponents/PostDialog.tsx (hunk 2 above)
Comment (Jack-Works) [reply src/components/InjectedComponents/PostDialog.tsx]: It's an ad-hoc code. will remove in future so not going to add it.

File: src/social-network-provider/twitter.com/ui/index.ts (hunk 3 above)
Comment (Jack-Works) [reply src/social-network-provider/twitter.com/ui/index.ts]: By providing a new i18n file?

Comment (guanbinrui) [reply src/social-network-provider/twitter.com/ui/index.ts]: A file only for overwriting fields?

**Pull request merged**